OpenSim/Tools/Test/vis_AppearanceTest.txt

```diff
@@ -0,0 +1,42 @@
+DecorativeFrame:1 bodyId:0 color:~[1,1,1] indexOnBody:0 Opacity:1 Rep:3 Scale:~[0.2,0.2,0.2] Transform:
+[1,0,0,0]
+[0,1,0,0]
+[0,0,1,0]
+[0,0,0,1]
+
+DecorativeSphere:1 bodyId:0 color:~[1,1,1] indexOnBody:0 Opacity:1 Rep:3 Scale:~[1,1,1] Transform:
+[1,0,0,0]
+[0,1,0,0]
+[0,0,1,0]
+[0,0,0,1]
+
+DecorativeFrame:1 bodyId:0 color:~[1,1,1] indexOnBody:0 Opacity:1 Rep:3 Scale:~[0.2,0.2,0.2] Transform:
+[1,0,0,1]
+[0,1,0,0]
+[0,0,1,0]
+[0,0,0,1]
+
+DecorativeSphere:1 bodyId:0 color:~[0,1,1] indexOnBody:0 Opacity:0.5 Rep:3 Scale:~[1,1,1] Transform:
+[1,0,0,1]
+[0,1,0,0]
+[0,0,1,0]
+[0,0,0,1]
+
+DecorativeFrame:1 bodyId:0 color:~[1,1,1] indexOnBody:0 Opacity:1 Rep:3 Scale:~[0.2,0.2,0.2] Transform:
+[1,0,0,2]
+[0,1,0,0]
+[0,0,1,0]
+[0,0,0,1]
+
+DecorativeFrame:1 bodyId:0 color:~[1,1,1] indexOnBody:0 Opacity:1 Rep:3 Scale:~[0.2,0.2,0.2] Transform:
+[1,0,0,3]
+[0,1,0,0]
+[0,0,1,0]
+[0,0,0,1]
+
+DecorativeSphere:1 bodyId:0 color:~[1,1,1] indexOnBody:0 Opacity:1 Rep:2 Scale:~[1,1,1] Transform:
+[1,0,0,3]
+[0,1,0,0]
+[0,0,1,0]
+[0,0,0,1]
+
```

Comment:
I realize that this is the "standard" file for the test, but corresponding this text file to the geometry in the model is challenging. Same goes for the previous test. Having tried to debug the visualization myself in the past (e.g. with changes to attach Geometry to Frames) I found it extremely time consuming to map the geometry in the model to corresponding text rendering. For example, why is everything `DecorativeFrame:1` or `DecorativeSphere:1`? Could you not use different bodies and transforms so that not everything is `bodyId:0` and what is `indexOnBody` and why is it always 0? if we can only see color as `[1, 1, 1]` then don't use use the SimTK::Color constants because again the correspondence is not obvious. You can use comments in test model creation block above to describe what you should see in the vis file. For example that `DecorativeGeometry::DrawWireframe` is `Rep:2` and the default (Shaded?) is `Rep:3`.  As a reviewer I am supposed to help verify that the standard is indeed correct, and I don't think I can do that given the potential ambiguities. That is another reason to use models that are created programmatically because you can document what the visualization should draw for each Component added to the Model.